src/gs-page.c

```diff
@@ -180,6 +184,9 @@ gs_page_app_installed_cb (GObject *source,
 		return;
 	}
 
+        now = g_date_time_new_now_local ();
+        g_settings_set (helper->settings, "install-timestamp", "x", g_date_time_to_unix (now));
+
 	/* only show this if the window is not active */
```

Comment:
This might look spurious, but we are setting the timestamp here because `gs_page_app_installed_cb` is also the kinda callback for `gs_page_update_app`.

But we might to take in consideration that this will **also** set the timestamp for something like `gs_page_install_app`